structopt-derive/src/attrs.rs

```diff
@@ -87,25 +150,54 @@ impl Attrs {
         use Meta::*;
         use NestedMeta::*;
 
-        let iter = attrs
+        let recognized_attrs = attrs
             .iter()
             .filter_map(|attr| {
                 let path = &attr.path;
-                match quote!(#path).to_string() == "structopt" {
-                    true => Some(
+                match quote!(#path).to_string().as_ref() {
+                    "structopt" | "casing" => Some(
```

Comment:
No, everything must be in the structopt attribute:
```rust
#[structopt(rename_all = "verbatim")]
struct Opt {}
```

Reply:
Should be no problem. Got distracted by the following part of your overview comment in #18.

> - manage #[rename_all(some_heck_casing)] on the struct or enum to change the default name conversion to the one asked, with a verbatim variant using the raw value.

Reply:
My bad